examples/systemcalls/systemcalls.c

```diff
@@ -93,7 +201,110 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
-    va_end(args);
+    openlog("do_exec_redirect_func",0,LOG_USER);
```

Comment:
@ceca5556 think about how you could avoid duplicating logic with the `do_exec` function.